src/components/publishing/__stories__/typography.story.tsx

```diff
@@ -1,15 +1,22 @@
 import { storiesOf } from "@storybook/react"
 import * as React from "react"
 
-import IconImageSet from "../icons/icon_imageset"
+import IconImageSet from "../icons/imageset"
+import IconRemove from "../icons/remove"
```

Comment:
#minor May also wanna decide to be consistent in singular/plural use of ‘icon’. E.g. `import IconImageSet from "../icon/imageset".